fe/fe-core/src/main/java/com/starrocks/connector/iceberg/CachingIcebergCatalog.java

```diff
@@ -139,12 +139,15 @@ public List<String> listTables(String dbName) {
     public Table getTable(String dbName, String tableName) throws StarRocksConnectorException {
         IcebergTableName icebergTableName = new IcebergTableName(dbName, tableName);
 
-        if (ConnectContext.get() == null || ConnectContext.get().getCommand() == MysqlCommand.COM_QUERY) {
+        if (ConnectContext.get().getCommand() == MysqlCommand.COM_QUERY) {
```

Comment:
should i do
        if (ConnectContext.get() != null && ConnectContext.get().getCommand() == MysqlCommand.COM_QUERY) 
to avoid null pointer exception?

Reply:
when the `ConnectContext.get()` is null, there are some background threads to refresh the table like mv or table auto refreshing